### PR TITLE
Make upload_symbols_to_crashlytics fail with a more descriptive error message.

### PR DESCRIPTION
**Pull request by lalunamel**
See https://github.com/fastlane/fastlane/issues/9477 for details.

### CODE REVIEW

File: fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb [hunk 1]
```diff
@@ -77,7 +77,7 @@ def self.find_api_token(params)
 
       def self.find_binary_path(params)
         params[:binary_path] ||= (Dir["/Applications/Fabric.app/**/upload-symbols"] + Dir["./Pods/**/upload-symbols"]).last
-        UI.user_error!("Please provide a path to the binary using binary_path:") unless params[:binary_path]
+        UI.user_error!("Failed to upload find Fabric's upload_symbols binary at /Applications/Fabric.app/**/upload-symbols or ./Pods/**/upload-symbols. Please specify the location of the binary explicitly by using the binary_path option") unless params[:binary_path]
```
Comment (ohayon) [inline fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb]: Failed to find Fabric's....

**Comment by lalunamel** [comment]
Looks like legal needs to be involved here.
Will be back in N days when that's resolved.

**Review by ohayon — CHANGES_REQUESTED** [review]
This is awesome! Thanks so much for making this PR @lalunamel!! Just one small change and we should be good to 🚀

**Review by ohayon — APPROVED** [review]

**Comment by lalunamel** [comment]
Howdy.
Legal says I've been added to the group CLA for my company, Pivotal, so you can remove the `cla: no` label.

**Comment by willnorris** [comment]
One of your commits is authored with your pivotal email, the other is with your gmail.  Please amend the second to also use your pivotal email.

**Comment by lalunamel** [comment]
There you go!

**Comment by willnorris** [comment]
sorry, one more minor thing.  could you add your pivotal.io [email to your github account](https://help.github.com/articles/adding-an-email-address-to-your-github-account/)? (Fortunately, all of this is just a one-time setup... future contributions should go much more smoothly, I promise)

**Comment by lalunamel** [comment]
done :)

**Comment by lalunamel** [comment]
nice job googlebot! what a nice robut. so halpful!

**Comment by ohayon** [comment]
Thanks for your patience and going through all of the CLA stuff @lalunamel !!! We really appreciate the change! 🚀

**Pull request merged**